provider/datagen/src/lib.rs

```diff
@@ -21,7 +21,8 @@
 //! use std::fs::File;
 //!
 //! fn main() {
-//!     DatagenProvider::default()
+//!     DatagenProvider::try_new(options::Options::default(), SourceData::latest_tested())
```

Comment:
Can we not provide a default implementation anymore?

Reply:
I don't feel that a "default" datagen provider is or will ever be a well-defined concept. Best to just have clients list out these arguments.

Besides that, `Default::default()` was causing problems because the default is now `LocaleInclude::Recommended` which the code currently assumes is converted to `LocaleInclude::Explicit`, but this can only be done in a `try_new` that has CLDR data available.

Reply:
👍